R/Private.R

```diff
@@ -115,3 +115,157 @@ nullToEmpty <- function(x) {
   x[is.null(x)] <- ""
   return(x)
 }
+
+
+
+makeDataExportable <- function(x,
```

Comment:
Can you add some unit tests for this function? This is useful as we want to know that all the checks are properly enforced